src/main.rs

```diff
@@ -154,6 +160,7 @@ pub async fn main() {
         async move {
             Ok::<_, Infallible>(service_fn(move |req: Request<Body>| {
                 let onion2 = Arc::clone(&onion1);
+                info!("{:?}", req);
```

Comment:
Clearly not sufficient as we don't log at least response status code. We have to comply with most http server access logs format. See [how warp is doing access logs](https://github.com/seanmonstar/warp/blob/fbbbc5b8377914463e4411ef6fc48bfd97d15a6d/src/filters/log.rs).